Source/WTF/wtf/PlatformUse.h

```diff
@@ -431,3 +431,9 @@
     || (PLATFORM(IOS_FAMILY) && !PLATFORM(IOS_FAMILY_SIMULATOR)))
 #define USE_MODERN_AVCONTENTKEYSESSION 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED < 150400) \
```

Comment:
Actually, since this quirk is only relevant on iPad, we should remove the MAC bit here -- we don't ever need to trigger this Quirk on macOS! That will save us doing these lookups for Mac users, too.